docs/api/cases/cases-api-set-default-connector.asciidoc

```diff
@@ -0,0 +1,232 @@
+[[cases-api-set-default-connector]]
+== Set default connector for cases API
+++++
+<titleabbrev>Set default connector</titleabbrev>
+++++
+
+Sets the default connector for cases.
+
+=== {api-request-title}
+
+`POST <kibana host>:<port>/api/cases/configure`
+
+`POST <kibana host>:<port>/s/<space_id>/api/cases/configure`
+
+=== {api-prereq-title}
+
+You must have `all` privileges for the *Cases* feature in the *Management*,
+*{observability}*, or *Security* section of the
+<<kibana-feature-privileges,{kib} feature privileges>>, depending on the
+`owner` of the cases.
+
+=== {api-description-title}
+
+Connectors are used to interface with external systems. Before you can set the
+default connector for your cases, you must create it.
+//Refer to <<register-connector>>.
+
+If you do not set a default connector, you can simply provide the connector
+details when you <<cases-api-create,create>> or <<cases-api-update,update>> a
+case.
+
+=== {api-path-parms-title}
+
+`<space_id>`::
+(Optional, string) An identifier for the space. If it is not specified, the
+default space is used.
+
+=== {api-request-body-title}
+
+`closure_type`::
+(Required, string) Specifies whether a case is automatically closed when it is
+pushed to external systems.
++
+--
+Valid values are:
+
+* `close-by-pushing`: Cases are automatically closed when they are pushed.
+* `close-by-user`: Cases are not automatically closed.
+--
+
+`connector`::
+(Required, object) An object that contains the connector configuration.
++
+.Properties of `connector`
+[%collapsible%open]
+====
+`fields`::
+(Required, object) An object containing the connector fields.
++
+--
+If you do not want a default connector, specify `null` for this object. If you
+want to omit any individual field, specify `null` as its value.
+
+////
+NOTE: Fields can be set but are not being used by case configuration. You can set the fields of the connector at <<cases-api-create>>.
```

Comment:
> NOTE: Fields can be set but are not being used by case configuration. You can set the fields of the connector at <<cases-api-create>>.

If this is still correct and these field settings are never used, should we just omit these details and recommend setting "fields" to "null"?

Reply:
Make sense to me. Yes, the fields are not being used at the moment.

Reply:
Thanks!  Done in fadfee7cca050f8de6f3d416e000a4ce539d8590